.circleci/config.yml

```diff
@@ -137,6 +137,7 @@ workflows:
           filters:
             branches:
               only: main
+      - check-api-compatibility
       - create-xcframework
```

Comment:
Looks like create-xcframework job was also updated to run checks?

Reply:
Right, but PR builds don't run `create-xcframework` by default.